src/Pelagos/Entity/Fileset.php

```diff
@@ -11,5 +12,36 @@
  */
 class Fileset extends Entity
 {
+    /**
+     * Collection of files.
+     *
+     * @var Collection
+     *
+     * @access protected
+     *
+     * @ORM\OneToMany(targetEntity="File", mappedBy="Fileset", cascade={"persist"}, orphanRemoval=true)
+     */
+    protected $files;
 
+    /**
+     * Getter for files.
+     *
+     * @return Collection
+     */
+    public function getFiles() : Collection
+    {
+        return $this->files;
+    }
+
+    /**
+     * Setter for files.
+     *
+     * @param Collection $files The collection of files in this fileset.
+     *
+     * @return void
+     */
+    public function setFiles(Collection $files) : void
```

Comment:
In additional, or maybe instead of this, I would add functions: addFile, removeFile that need the required fields to make a file a file. Or else you can just add files, without any attributes.

Reply:
Rename it to addFiles instead?

Reply:
I was thinking adding one file at the time, but there could be a add multiple files one too?

Reply:
I thought we can create all the files and then associate the collection to the Fileset entity

Reply:
I think this is ok for now, things might change when we get further along.